### DISCUSSION TITLE
Option to disable FQDN option on applications

### DISCUSSION THREAD

**Discussion by dgc1980** [coollabsio/coolify#3876] [Feature Requests]
a lot of my applications are python bots that do not need a web interface so the option for a web proxy is not needed, so a option to disable the FQDN completely would be great

**Comment by c43721** [comment]
I +1 this. I am coming from Caprover where this is an option (the "do not expose as a webapp" option) and this would be nice.

It works fine without it as I tested, but it would be really cool to have it baked into the app as well instead of this weird kinda bug like behavior (where sometimes the application logs don't show, the build logs won't show until you save, ect)

**Comment by andrasbacsai** [comment]
This is already implemented. You can set an application to be a `bot`. That means it does not requires any domain.

**Comment by centralhardware** [comment]
@andrasbacsai can you please explain how to set an applicatoin to be a bot?

**Comment by dgc1980** [comment]
> @andrasbacsai can you please explain how to set an applicatoin to be a bot?

![image](https://github.com/user-attachments/assets/c19e8717-9a88-4068-a791-24c8213de2b3)

while creating the application after selecting the git repo and docker location, you can enable it is a bot there.

**Reply by centralhardware** [reply]
Looks like, latest 4 beta version doesn't have this option
